ansible_collections/arista/cvp/docs/schema/cv_change_control_v3.md

```diff
@@ -0,0 +1,23 @@
+# Schema for cv_change_control_v3
+
+| Variable | Type | Required | Default | Choices | Description |
+| -------- | ---- | -------- | ------- | ------------------ | ----------- |
+| change_id | list | No |  |  | List of change IDs to get/remove |
+| name | str | No |  |  | The name of the change control. If not provided, one will be generated |
```

Comment:
Mixing sentence and non sentence structure here. Probably not a big deal but thought you should be aware.

Reply:
Feel free to make suggestions.